google_guest_agent/oslogin.go

```diff
@@ -99,15 +99,18 @@ func (o *osloginMgr) set(ctx context.Context) error {
 		logger.Infof("Disabling OS Login")
 	}
 
-	if err := writeSSHConfig(enable, twofactor, skey); err != nil {
+	// Unstable configuration section is a section with no guarantee of long term compatibility/support.
+	pamlessAuthStack := config.Section("Unstable").Key("pamless_auth_stack").MustBool(false)
```

Comment:
do we also want to add section for this in `instance_configs.cfg` with default/empty value?

Reply:
The default value is set to disabled, so it makes sense only to have config entries on testing deployments that will either happen manually or by deriving images, so I don't think we need.